auraed/src/init/system_runtimes/pid1_system_runtime.rs

```diff
@@ -87,9 +87,6 @@ impl SystemRuntime for Pid1SystemRuntime {
         // TODO We likely to do not need to mount these filesystems.
         // TODO Do we want to have a way to "try" these mounts and continue without erroring?
 
-        MountSpec { source: None, target: "/dev", fstype: Some("devtmpfs") }
-            .mount()?;
```

Comment:
This was throwing a "Resource busy" so I presumed this wasn't actually needed?

Reply:
Is it just a feature of cloud-hypervisor that /dev is already mounted, but we would need to mount it in a non-vm context (i.e., true-true PID 1 for lack of a better identifier)?

Maybe the try and continue approach is right?

Reply:
Looking at [cloud-hypervisor's config](https://raw.githubusercontent.com/cloud-hypervisor/cloud-hypervisor/main/resources/linux-config-x86_64) that you linked to in #439 I see the following in the file:

```
CONFIG_DEVTMPFS=y
CONFIG_DEVTMPFS_MOUNT=y
```

Would this config be causing the mount to already be mounted?

Reply:
Probably so. "Resource busy" usually means that the device is already mounted afaik

Reply:
sounds like we should try and mount just in case the config doesn't exist to automatically mount it (but allow the error if it fails... maybe log a warning)

Reply:
We should probably do that for all mounts we deem essential. I'm wondering though if we should do this now or if we can postpone this to later when we allow for kernels other than the ones we control - if we ever are planning to do this at all, as this introduces a whole set of other challenges. Until that point we know with 100% certainty what is mounted and what is not.

Reply:
Lets postpone until later once we have better control of the kernels we are running.